test/unit/uk/gov/hmrc/apiplatformtest/controllers/PrivilegedApiControllerSpec.scala

```diff
@@ -34,7 +34,7 @@ class PrivilegedApiControllerSpec extends AsyncHmrcSpec with AuthTestSupport wit
     val underTest = new PrivilegedApiController(mockAuthConnector, stubControllerComponents())
   }
 
-  // implicit val timeout = akka.util.Timeout(defaultTimeout)
+  // implicit val timeout = org.apache.pekko.util.Timeout(defaultTimeout)
```

Comment:
probably fine to just remove this comment rather than update it

Reply:
I'll remove it. I left it because the person who put it there never leaves unnecessary comments, so I was thinking it's a debugging aid :thinking:  Probably not.